Doc/reference/lexical_analysis.rst

```diff
@@ -671,8 +672,9 @@ The parts of the string outside curly braces are treated literally,
 except that any doubled curly braces ``'{{'`` or ``'}}'`` are replaced
 with the corresponding single curly brace.  A single opening curly
 bracket ``'{'`` marks a replacement field, which starts with a
-Python expression.  After the expression, there may be a conversion field,
-introduced by an exclamation point ``'!'``.  A format specifier may also
+Python expression. To display both the expression text and its value after evaluation,
+(useful in debugging), an equals ``'='`` may be added after the expression. A conversion field,
```

Comment:
"equals" -> "equal sign" everywhere (except in the index entries, since it's spelled "equals" in other index entries for the same character).

Also please reflow the lines you changed to stay within ~80 characters.